client/assets/js/admin-controller.js

```diff
@@ -137,5 +145,148 @@ angular.module('portal').controller('adminController', function ($scope, $http,
             // needs implementation
         });
     };
-    
+
+    // get all admin
+    $http.get('/api/user/getUsers').then(function(response) {
+        $scope.admins = {};
+        $scope.comUsers = {};
+        $scope.userData = response.data;
+        console.log("ALL USERS: ", $scope.userData);
+        $scope.userData.forEach(function(user) {
+            if (user.isAdmin === true) {
+                $scope.admins[user.email] = user.name;
+            } else {
+                $scope.comUsers[user.email] = user.name;
+            };
+        });
+    });
+
+
+    $scope.addAdmin = function() {
+        $scope.internalError = false;
+
+        if ($scope.comUsers[$scope.changeAdminEmail]) {
+
+            $scope.changeUser = $http.post('/api/user/getSingleUser', {
```

Comment:
shouldn't this be a GET request instead of a POST request? also, i believe you can use the route `/api/user` instead